### ISSUE TITLE
Question: Stratification in Beale Ratio Estimator 

### ISSUE THREAD

**Issue by aspaetzel** [VeronicaNava/RiverLoad#1]
Do the Beale Ratio Estimator (BRE) functions in RiverLoad use stratification? I read the citation for the method (Quilbe et al., 2006), and the article discusses BRE with and without stratification.

**Comment by VeronicaNava** [comment]
No, the function not directly implement stratification options. But for seasonal stratification, you can first subset your dataset and then run the algorithm separately on the subsets
